demo/index.js

```diff
@@ -47,15 +47,18 @@ function embedContent(payload) {
 		? revSdk.embedVideo('#embed', videoId, embedConfig)
 		: revSdk.embedWebcast('#embed', webcastId, embedConfig);
 
+	globalThis.vbrickPlayer = currentEmbed;
```

Comment:
This looks like something from Luke's pr

Reply:
yep, I'll remove it if his gets merged first